ruby/example_code/dynamodb/GettingStarted/MoviesDeleteTable.rb

```diff
@@ -0,0 +1,46 @@
+# snippet-sourcedescription:[MoviesDeleteTable.rb demonstrates how to ]
```

Comment:
I'll add this to the Ruby section of the repo as dynamodb_ruby_delete_movies_table.rb.

Reply:
please, don't change or add anything based on this pull request it is closed until we can sort out the logistics and the goals of the release